nexus/src/app/ip_pool.rs

```diff
@@ -28,16 +28,19 @@ impl super::Nexus {
         opctx: &OpContext,
         new_pool: &params::IpPoolCreate,
     ) -> CreateResult<db::model::IpPool> {
-        self.db_datastore.ip_pool_create(opctx, new_pool, None).await
+        self.db_datastore
+            .ip_pool_create(opctx, new_pool, /* internal_only= */ false)
+            .await
     }
 
     pub async fn ip_pool_services_create(
```

Comment:
It's outside the scope of this PR, but I wonder if _IP Pool Services_ is the right way to frame this now. Should it just be internal ip pools?

Reply:
I hear you on this; maybe `s/service/internal` would be more consistent. When I originally wrote this, I think I was writing "service" as shorthand for "internal service".